megamek/src/megamek/client/ui/swing/BoardEditor.java

```diff
@@ -1752,21 +1741,39 @@ public void actionPerformed(ActionEvent ae) {
         } else if (ae.getSource().equals(texTerrainLevel)) {
             updateWhenSelected();
         } else if (ae.getSource().equals(texTerrExits)) {
+            int exitsVal = texTerrExits.getNumber();
+            if (exitsVal == 0) {
+                cheTerrExitSpecified.setSelected(false);
+            } else if (exitsVal > 63) {
+                texTerrExits.setNumber(63);
+            }
             updateWhenSelected();
         } else if (ae.getSource().equals(butTerrExits)) {
             ExitsDialog ed = new ExitsDialog(frame);
-            cheTerrExitSpecified.setSelected(true);
-            ed.setExits(texTerrExits.getNumber());
+            int exitsVal = texTerrExits.getNumber();
+            ed.setExits(exitsVal);
             ed.setVisible(true);
-            texTerrExits.setNumber(ed.getExits());
+            exitsVal = ed.getExits();
+            texTerrExits.setNumber(exitsVal);
+            if (exitsVal != 0) {
+                cheTerrExitSpecified.setSelected(true);
```

Comment:
How about cheTerrExitSpecified.setSelected(exitsVal != 0)

Reply:
and on line 1772

Reply:
Fixed